packages/launchpad/src/checklist-item/index.tsx

```diff
@@ -38,28 +39,47 @@ const ChecklistItem: FC< Props > = ( { task, isPrimaryAction, onClick } ) => {
 		disabled,
 		...( disabled ? {} : { href: buttonHref } ),
 	};
+
 	const onClickHandler = onClick || actionDispatch;
+	const isClickable = !! ( buttonProps.href || onClickHandler );
+
+	// Display chevron if task is incomplete.
+	// Don't display chevron and badge at the same time.
+	// Don't display chevron if item is not clickable.
+	const shouldDisplayChevron = ! completed && ! disabled && ! task.badge_text && isClickable;
```

Comment:
These 2 variables just changed the position in the function.
The `shouldDisplayChevron` was moved to have the `isClickable` available, and the `shouldDisplayTaskCounter` was moved together because they are similar variables.